pre-deploy.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+'use strict';
+
+const {promisify} = require('util');
+const execFile = promisify(require('child_process').execFile);
+const fse = require('fs-extra');
+const glob = promisify(require('glob'));
+
+/**
+ * grpc comes with a native extension that is built upon install. However,
+ * unless we happen to be deploying from the exact same architecture that we are
+ * deploying to, we need to recompile it for App Engine. This should be a
+ * non-issue once App Engine runs `npm install` for us.
```

Comment:
Can we check this is necessary still? Deploy should now be running `npm install` and will respect our `package-lock.json`.